src/transformers/pipelines/text_classification.py

```diff
@@ -131,6 +132,19 @@ def __call__(self, *args, **kwargs):
 
     def preprocess(self, inputs, **tokenizer_kwargs) -> Dict[str, GenericTensor]:
         return_tensors = self.framework
+        if isinstance(inputs, dict):
+            return self.tokenizer(**inputs, return_tensors=return_tensors, **tokenizer_kwargs)
+        elif isinstance(inputs, list) and len(inputs) == 1 and isinstance(inputs[0], list) and len(inputs[0]) == 2:
+            # Manual BC use case where passing list of list of list is understood a proper text pair.
```

Comment:
```suggestion
            # It used to be valid to use a list of list of list for text pairs, keeping this path for BC
```
I did not understand the comment as it is.